### PR TITLE
Ltp 13 program registration page

### PR DESCRIPTION
**Pull request by Kodoh**
registration page added plus some basic validation done

### CODE REVIEW

File: register.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <div class="form-group">
+                <div class="group">
+              <label class="pass" for="passInp">Password</label>
+            </div>
+              
+              <input type="password" class="form-control" id="passwordInput">
+            
+              <br>
+              <span id="StrengthDisp" class="badge displayBadge"></span>
+              <script src="form.js"></script>
```
Comment (Kodoh) [inline register.html:34]: To catch the DOM elements in the form

**Review by nicmiel — APPROVED** [review]

File: register.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+              <label class="pass" for="passwordInput">Password</label>
+            </div>
+              
+              <input type="password" class="form-control" id="passwordInput" name ="passwordInput" required>
+            </div>
+            <div class="form-group">
+                <div class="group">
+              <label class="pass" for="passwordMatchInput">Confirm Password</label>
+            </div>
+            <input type="password" class="form-control" id="passwordMatchInput" name = "passwordMatcbInput" required>
```
Comment (Benzene06) [inline register.html]: Correct the name

Comment (Kodoh) [reply register.html]: good spot! will change that but will have to add it for the next PR as this has been merged already

Comment (Kodoh) [reply register.html]: Actually thats from an old commit and was addressed in the final commit hence has been resolved good spot anyway

Comment (Benzene06) [reply register.html]: Ohhh I see. Good good

**Pull request merged**